File: plugins/aptana/Wrap with Abbreviation.js
/*
 * Menu: Zen Coding > Wrap with Abbreviation
 * Kudos: Sergey Chikuyonok (http://chikuyonok.ru)
 * License: EPL 1.0
 * Key: M1+M2+A
 * DOM: http://download.eclipse.org/technology/dash/update/org.eclipse.eclipsemonkey.lang.javascript
 * 
 * @include "/EclipseMonkey/scripts/monkey-doc.js"
 * @include "lib/core.js"
 * @include "settings.js"
 */

// init engine
try {
	include('my_zen_settings.js');
} catch(e){}

include('zencoding.js');

function main() {
	zen_editor.setContext(editors.activeEditor);
	var abbr = prompt('Enter abbreviation:');
	if (abbr)
		zen_coding.runAction('wrap_with_abbreviation', zen_editor, abbr);
}

File: plugins/e2/Zen Coding.tmbundle/Support/wrap_with_abbreviation.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from zencoding import zen_core as zen_coding
from zen_editor import ZenEditor

editor = ZenEditor()
abbr = editor.prompt('Enter abbreviation')
if abbr is not None:
	zen_coding.run_action('wrap_with_abbreviation', editor, abbr)